app/views/projects/new.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Projects#new</h1>
+<p>Find me in app/views/projects/new.html.erb</p>
```

Comment:
This should be deleted since its namespace will be different when it gets written.